src/pages/GamePage/components/InfoSkills.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { attributes, skills } from '../../../utils/attributesSkills';
+import Skill from './Skill';
+
+export default function InfoSkills({ stats }) {
+  return (
+    <div className="info__skills-wrapper">
```

Comment:
Nested map functions aren't so cool. You could wrap it in a function like attributes.map(attribute => renderSkillsFrom(attribute))